wire-ios/Wire-iOS Tests/DeviceManagement/DeviceDetailsViewTests.swift

```diff
@@ -23,13 +23,23 @@ import WireSyncEngineSupport
 import WireRequestStrategySupport
 
 final class DeviceDetailsViewTests: BaseSnapshotTestCase, CoreDataFixtureTestHelper {
+
+    private let mockProteusId: String = "abcdefghijklmnop"
```

Comment:
**question: ** Looks like we are repeating same mocks in every class

Reply:
It's only two places and a simple string - I don't see any problems and the strings can be even different for each test, e.g. you want to have an empty string for one test case.